lib/xclarity_client/services/osimage_management.rb

```diff
@@ -25,21 +25,21 @@ def import_osimage(serverId, path)
  
         msg="inputs serverId=#{serverId},path=#{path}"
         $lxca_log.info self.class.to_s+" "+__method__.to_s, msg
-        opts = { "Action": "Init" }
+        opts = { "Action" => "Init" }
```

Comment:
The correct use of a hashrocket syntax should use symbols.
So it would look like `:Action => "Init"`

Reply:
@felipedf I have updated code..Thanks

Reply:
@umrikar Yeah, you are correct, the point is consistency ( which is ironic because our client doesn't have much ), but we will try to implement more rules on rubocop and codeclimate to address issues like that, we mostly use hashrocket style, you can see it here http://www.rubydoc.info/gems/rubocop/0.45.0/RuboCop/Cop/Style/HashSyntax